### DISCUSSION TITLE
Different types of Diesel?

### DISCUSSION THREAD

**Discussion by AndresSepulveda** [OpenDrift/opendrift#1058] [Q&A] [answered]
Hi,

I´ve noticed there are several types of oil_types in the OpenOil model, namely

'GENERIC DIESEL',
'DF2 SUMMER (DIESEL), TESORO', 
'DF2 WINTER (DIESEL)',
'DIESEL', 
'DIESEL FUEL OIL (ALASKA)', 
'DIESEL FUEL OIL (ALASKA)',                 ** defined twice **
'DIESEL FUEL OIL (CANADA)', 
'DIESEL FUEL OIL (SOUTHERN USA 1994)', 
'DIESEL FUEL OIL (SOUTHERN USA 1997)', 
'DIESEL FUEL OIL NO.2 (BONDED), TESORO', 
'DIESEL/HEATING OIL NO.2, CHEVRON',

plus 'BIODIESEL'

Where can I find their characteristics, in order to understand which is the one used as "Diesel" here in Chile?

**Comment by gauteh** [comment, marked as answer]
Hi Andres,

All oils should now be present in: https://adios.orr.noaa.gov/oils . We
ship a snapshot of these, but they do not change much.

The simulation is usually not very sensitive to choosing slightly different
oils.

Regards, Gaute

lør. 28. jan. 2023 kl. 00:51 skrev Andres Sepulveda <
***@***.***>:
